templates/index.template.html

```diff
@@ -66,12 +66,11 @@ <h2 class="display-3 text-white">Open-source enthusiasts</h2>
         </div>
         <div class="col-md-8 order-1">
           <div class="p-5">
-            <h2 class="display-4">Who we are</h2>
-            <p>Inaka is a community of open-source enthusiasts (developers, PMs, devops, QA experts, etc.).</p>
-            <p>We maintain all the repos listed below (and some others, too).</p>
-          </div>
+              <h2 class="display-4">Who we are?</h2>
```

Comment:
The `?` was intentionally removed before.

Reply:
There was some mistake regarding some changes on the template and in the index.html. It should be ok now.